source/class/cv/TemplateEngine.js

```diff
@@ -274,16 +274,17 @@ qx.Class.define('cv.TemplateEngine', {
      * Initialize the {@link cv.io.Client} for backend communication
      */
     initBackendClient: function () {
-      let backendName = cv.Config.configSettings.backend || cv.Config.backend;
-      let backendUrl = cv.Config.configSettings.backendUrl || cv.Config.backendUrl;
+      let backendName = cv.Config.backend || cv.Config.configSettings.backend || 'default';
+      const backendUrl = cv.Config.backendUrl || cv.Config.configSettings.backendUrl;
+      const backendLoginUrl = cv.Config.backendLoginUrl || cv.Config.configSettings.backendLoginUrl;
       const mapping = {
         oh: 'openhab',
         oh2: 'openhab2'
       };
       if (Object.prototype.hasOwnProperty.call(mapping, backendName)) {
         backendName = mapping[backendName];
       }
-      this.visu = cv.Application.createClient(backendName, backendUrl);
+      this.visu = cv.Application.createClient(backendName, backendLoginUrl, backendUrl);
```

Comment:
This change breaks the openHAB-backend client. This one does not need a special loginURL because it uses an authentication token send in the header, but it needs the backendURL only.

I am not sure if we need to distinguish between a loginUrl and a backendUrl, because none of the backends need both of them. So I don't see the need for this, as there is no benefit and it makes things just more complicated.